pymc3/variational/inference.py

```diff
@@ -408,6 +446,78 @@ def from_mean_field(cls, mean_field, gpu_compat=False):
 
     @classmethod
     def from_advi(cls, advi, gpu_compat=False):
+        """
+        Construct FullRankADVI from ADVI
+
+        Parameters
+        ----------
+        advi : ADVI
+
+        Flags
+        -----
+        gpu_compat : bool
+            use GPU compatible version or not
+
+        Returns
+        -------
+        FullRankADVI
+        """
         inference = cls.from_mean_field(advi.approx, gpu_compat)
         inference.hist = advi.hist
         return inference
+
+
+def approximate(n=10000, local_rv=None, method='advi', model=None, **kwargs):
```

Comment:
I think `fit` is probably appropriate (and shorter than `approximate`). 

I would also suggest keeping `ADVI` (and other acronyms) upper case, again for consistency. 

Sorry to be picky about these little things, but once we make interface decisions they are hard to undo if we don't like them later, so its worth spending the time to think about them now.